testsuite/testcases/src/load_vs_perf_benchmark.rs

```diff
@@ -42,6 +43,17 @@ impl Workloads {
         }
     }
 
+    fn num_phases(&self, index: usize) -> usize {
+        match self {
+            Self::TPS(_) => 1,
+            Self::TRANSACTIONS(workloads) => if workloads[index].is_phased() {
```

Comment:
nit: Not an issue with this PR, but I had to look up `is_phased()` to determine why some workloads get 1 phase vs. 2 phases. It makes me wonder if we'd be better off changing this method name, or adding a new method to the workload itself? e.g., `num_phases`, which hides this logic? But, no biggie.